systemsdk/easy-log-bundle/1.10/config/packages/dev/easy_log.yaml

```diff
@@ -0,0 +1,7 @@
+easy_log:
+    log_path: '%kernel.logs_dir%/%kernel.environment%-readable.log'
+    # Use below if you're running application in container and want to log in stdout
+    #log_path: 'php://stdout'
+    max_line_length: 120
+    prefix_length: 2
+    ignored_routes: ['_wdt', '_profiler']
```

Comment:
You dont need to show all your config. Just keep sensible defaults.

Reply:
It is already sensible defaults (we usually don't need to log debug info)

Reply:
I suggest you keep: 

```yaml
easy_log:
    log_path: '%kernel.logs_dir%/%kernel.environment%-readable.log'
    ignored_routes: ['_wdt', '_profiler']
 ```

The rest can be removed. There is no need to show all your config. 

Feel free to add a link to your documentation if you want to.